cnxarchive/sql/search/author.part.sql

```diff
@@ -17,7 +17,7 @@ WHERE
    OR
    u.last_name ~* req(%({0})s::text)
    OR
-   u.full_name ~* req(%({0})s::text)
+   u.full_name ~* regexp_replace(req(%({0})s::text), ' ', '.*.')
```

Comment:
Come to think about it, should we use `.+` instead?  To make sure when people search for `jill miller` we won't return `jillmiller`? (just in case)